src/com/twu/biblioteca/MainMenu.java

```diff
@@ -31,6 +33,10 @@ public void manageOptions() {
                 showList();
                 counter = 1;
             }
+            else if(input.equals("2")) {
```

Comment:
Out of curiosity, what do you think would happen if the user entered " 2" or "2 " or something like that?  Is that something you could/should handle gracefully?

Reply:
Currently the code accepts it but it was not planned by me and I should have accounted for that by cleaning up the user input. I will add a trim method to clean up the user input.